packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/PageContent.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  Field,
+  RichText as JssRichText,
+  useSitecoreContext,
+  LayoutServiceContext,
+  RouteData
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+export type SitecoreContextValue = LayoutServiceContext & {
+  itemId?: string;
+  route: RouteData;
+};
+
+interface Fields {
+  Content: Field<string>;
+}
+
+type PageContentProps = {
+  params: { [key: string]: string };
+  fields: Fields;
+};
+
+const ComponentContent = (props: any) => {
+  return (
+    <div className={`component content ${props.styles?.replace(/\|/g, ' ')}`}>
+      <div className="component-content">
+        <div className="field-content">
+          {props.children}
+        </div>
+      </div>
+    </div>  
+  )
+}
+
+export const Default = (props: PageContentProps): JSX.Element => { 
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  if (!(props.fields && props.fields.Content) && !sitecoreContext.route.fields.Content) {
+    return (
+      <div className={`component content ${props.params.styles?.replace(/\|/g, ' ')}`}>
+        <div className="component-content">
+          <div className="field-content">
+            [Content]
+          </div>
+        </div>
+      </div>  
+    )
+  }
+  
+  if (props.fields && props.fields.Content) {
```

Comment:
In both `if` and `else` statements you have identical components but with different values provided for `RichText`.
You can create a single component which you will return:
```
const field = props.fields && props.fields.Content ? props.fields.Content : sitecoreContext.route.fields.Content

    return (
      <ComponentContent styles={props.params.styles}>
        <JssRichText field={field} />
      </ComponentContent>
    );
```